rpmconf/Spec/zimbra-imap.spec

```diff
@@ -0,0 +1,36 @@
+#
+# spec file for zimbra-imap.rpm
+#
+Summary: Zimbra IMAP
+Name: zimbra-imap
+Version: @@VERSION@@
+Release: @@RELEASE@@
+License: Various
+Group: Applications/Messaging
+URL: http://www.zimbra.com
+Vendor: Zimbra, Inc.
```

Comment:
This looks fine to me, but we need to change Zimbra, Inc. to Synacor Inc to all spec files and general info but doesn't affect functionally as we all know.